update/ansible/playbook/tasks/roles/grafana/files/grafana.ini

```diff
@@ -4,11 +4,29 @@
 [paths]
 # Directory where grafana will automatically scan and look for plugins
 plugins = /srv/grafana/plugins
+# Directory where grafana can store logs
+logs = /srv/logs
+# Path to where grafana can store temp files, sessions, and the sqlite3 db (if that is used)
+data = /srv/grafana
+
+#################################### Logging ##########################
+[log]
+# Either "console", "file", "syslog". Default is console and file
+mode = console
+
+# For "console" mode only
+[log.console]
+# log line format, valid options are text, console and json
+format = console
 
 #################################### Server ####################################
 [server]
 # enable gzip
 enable_gzip = true
+# The public facing domain name used to access grafana from a browser
+domain = 127.0.0.1
```

Comment:
Will CLI arg overwrite this value? Cause we have domain setting on the UI.

Reply:
> Will CLI arg overwrite this value? Cause we have domain setting on the UI.

Short answer is yes.
Long answer: managed is basically generating a new `/etc/supervisord.d/grafana.ini` file, which launches `grafana server` with a number of what is called [configuration overrides](https://grafana.com/docs/grafana/latest/cli/#override-a-configuration-setting). The latter will take precedence over settings found in `/etc/grafana/grafana.ini`.